dotCMS/src/integration-test/java/com/dotcms/util/MimeTypeUtilsTest.java

```diff
@@ -25,6 +26,122 @@ public static void prepare() throws Exception {
         IntegrationTestInitService.getInstance().init();
     }
 
+    @Test
+    public void test_match_MimeType_all() throws IOException {
+
+        boolean match = MimeTypeUtils.match("*/*", "image/jpeg");
+
+        Assert.assertTrue("* must match with image/jpeg", match);
+
+        match = MimeTypeUtils.match("*/*", "application/pdf");
+
+        Assert.assertTrue("* must match with application/pdf", match);
+
+        match = MimeTypeUtils.match("*/*", "text/plain");
+
+        Assert.assertTrue("* must match with text/plain", match);
+    }
+
+    @Test
+    public void test_match_MimeType_partial_image() throws IOException {
+
+        boolean match = MimeTypeUtils.match("image/*", "image/jpeg");
+
+        Assert.assertTrue("image/* must match with image/jpeg", match);
+
+        match = MimeTypeUtils.match("image/*", "image/webp");
+
+        Assert.assertTrue("image/* must match with image/webp", match);
+
+        match = MimeTypeUtils.match("image/*", "image/png");
+
+        Assert.assertTrue("image/* must match with image/png", match);
+
+        match = MimeTypeUtils.match("image/*", "application/vnd.hzn-3d-crossword");
+
+        Assert.assertFalse("image/* must not match with application/vnd.hzn-3d-crossword", match);
+    }
+
+    @Test
+    public void test_match_MimeType_partial_application() throws IOException {
+
+        boolean match = MimeTypeUtils.match("application/*", "application/vnd.hzn-3d-crossword");
+
+        Assert.assertTrue("application/* must match with application/vnd.hzn-3d-crossword", match);
+
+        match = MimeTypeUtils.match("application/*", "application/x-7z-compressed");
+
+        Assert.assertTrue("application/* must match with application/x-7z-compressed", match);
+
+        match = MimeTypeUtils.match("application/*", "application/pdf");
+
+        Assert.assertTrue("application/* must match with application/pdf", match);
+
+        match = MimeTypeUtils.match("application/*", "image/png");
+
+        Assert.assertFalse("application/* must not match with image/png", match);
+    }
+
+    @Test
+    public void test_match_MimeType_partial_text() throws IOException {
+
+        boolean match = MimeTypeUtils.match("text/*", "text/x-asm");
+
+        Assert.assertTrue("text/* must match with text/x-asm", match);
+
+        match = MimeTypeUtils.match("text/*", "text/css");
+
+        Assert.assertTrue("text/* must match with text/css", match);
+
+        match = MimeTypeUtils.match("text/*", "text/csv");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "text/csv" appears 4 times in this file; the first occurrence is on line 95](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5028307)